Assets/HoloToolkit/Utilities/Scripts/ManualCameraControl.cs

```diff
@@ -37,6 +37,16 @@ public enum MouseButton
     private bool appHasFocus = true;
     private bool wasLooking = false;
 
+    //changed the input axes strings to variables exposing them to the editor for non-standard 
```

Comment:
Please match comments formatting with other comments. Also perhaps a ToolTip for the fields so devs know they can change these mapping in ProjectSettings->Input if they want to.

Else, looks like a good change.